cli/cli.go

```diff
@@ -96,15 +96,15 @@ func (c *cli) OnInitialize(initializers ...func()) {
 }
 
 // ClientConn returns the grpc connection to the API.
-func (c *cli) ClientConn() (*grpc.ClientConn, error) {
+func (c *cli) ClientConn() *grpc.ClientConn {
 	if c.clientConn == nil {
 		var err error
 		c.clientConn, err = NewClientConn(c.Server(), GetToken())
 		if err != nil {
-			return nil, err
+			c.Console().Fatalf("error while establishing grpc connection : %s", err)
```

Comment:
`Fatalf` will prepend 'Error' to the message. Maybe `unable to establish grpc connection: %s\n` is better (don't forget newline!)  ;)